### PR TITLE
Handle Close error

### LINKED ISSUE #6: File close error not checked
https://github.com/mrunalp/fileutils/blob/master/fileutils.go#L65 should check the error returned from `Close()` as it may flush bytes to storage and encounter a failure.

### PR DESCRIPTION
**Pull request by ash2k**
Fixes #6.

@kolyshkin

### CODE REVIEW

File: fileutils.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func CopyFile(source string, dest string) error {
 
 	uid := int(st.Uid)
 	gid := int(st.Gid)
-	modeType := si.Mode()&os.ModeType
+	modeType := si.Mode() & os.ModeType
```
Comment (kolyshkin) [inline fileutils.go:25]: NB: this is from `gofmt` (maybe a change in a recent version)

**Review by kolyshkin — APPROVED** [review]
LGTM

**Comment by kolyshkin** [comment]
@mrunalp PTAL

**Review by mrunalp — APPROVED** [review]
LGTM

**Pull request merged**